xchange-bitmex/src/test/java/org/knowm/xchange/bitmex/service/marketdata/BitmexOrderBookFetchIntegration.java

```diff
@@ -0,0 +1,43 @@
+package org.knowm.xchange.bitmex.service.marketdata;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.knowm.xchange.ExchangeFactory;
+import org.knowm.xchange.bitmex.BitmexExchange;
+import org.knowm.xchange.currency.CurrencyPair;
+import org.knowm.xchange.dto.marketdata.OrderBook;
+import org.knowm.xchange.service.marketdata.MarketDataService;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class BitmexOrderBookFetchIntegration {
+
+  public static MarketDataService marketDataService;
+  public static BitmexExchange bitmexExchange;
+
+  @Before
+  public void setUp() {
+    bitmexExchange =
+        (BitmexExchange) ExchangeFactory.INSTANCE.createExchange(BitmexExchange.class.getName());
+    marketDataService = bitmexExchange.getMarketDataService();
+  }
+
+  @Test
+  public void getOrderBookTest() throws IOException {
+    CurrencyPair pair = new CurrencyPair("XBT", "M19");
```

Comment:
Another expiring contract reference. Can we update this test to use `determineActiveContract`?